pysb/bng.py

```diff
@@ -770,6 +776,68 @@ def _parse_netfile(model, lines):
         pass
 
 
+def _parse_parameter(model, line):
+    index, pname, pval, hash, ptype = line.strip().split()
+    par_names = model.components.keys()
+    if pname not in par_names:
+        if ptype == 'Constant' and pname not in model._derived_parameters.keys():
+            try:
+                p = pysb.core.Parameter(pname, pval, _export=False)
```

Comment:
This issue doesn't seem to appear in the test suite, even when I tried fudging around with various forms. Anything with operators should, in theory, appear as a `ConstantExpression` instead of `Constant`, anyway. In the absence of a verifiable test case, I'll just remove this check.